docs/faq.md

```diff
@@ -62,3 +63,9 @@ This makes the gene expression values broadly comparable, but doesn't explicitly
 If the scientific question and analysis methods require datasets to be batch corrected, users should first investigate the existence of batch effects using methods such as Principal Components Analysis.
 If the source dataset is associated with major sources of variability in the data, users may wish to use a meta-analysis framework considering each dataset independently or to apply a batch correction tool.
 It may be sufficient to include batch, dataset, or platform as covariates for certain analyses.
+
+#### Why are the expression values different if I regenerate a dataset?
+
+The `Regenerate Files` button triggers the creation of a new dataset using the same options and including the same samples as the original, expired dataset.
+If there are any changes to the way we process or aggregate datasets between the initial dataset creation and regeneration, these will be reflected in the new, regenerated dataset and may result in different values.
+Users should take this into account when managing datasets obtained from refine.bio.
```

Comment:
Do we want to suggest that folks archive? Also, should we mention the target quantiles (Do we provide these in the download package? Should we?).

Reply:
> Also, should we mention the target quantiles (Do we provide these in the download package? Should we?).

We do not currently include this, but it could be helpful to include the target quantiles in downloads. The way I would mention this given our current implementation would be something like:
```
If there are any changes to the way we process or aggregate datasets 
(e.g., generate [new target quantiles](http://docs.refine.bio/en/latest/main_text.html#quantile-normalization)) 
between the initial dataset creation and regeneration, ...
```

Reply:
> Do we want to suggest that folks archive the downloaded data?

Do you have a specific method in mind? Tracking with a version control system, for example, may be at odds with the T&C depending on how it's done.